zachsinterview/src/main/java/com/imprev/zachsinterview/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.imprev.zachsinterview;
+
+public class Main {
+
+    /**
+     * Demonstrates how to connect to thirdPartyAPI and tests how the numerical key store may be used.
+     */
+    public static void main(String[] argv) throws NumberStoreException {
+        
+        String username = "moxiworks";
+        String password = "fjdskal3kd!jdkdInterviewFun!!!";
+        String apiUrl = "https://example.com/thirdPartyApiURL";
+        
+        NumberStore numberStore = new NumberStoreImpl(username, password, apiUrl , 80);
+        NumberStoreAdditionalOperations keyFun = new NumberStoreAdditionalOperationsImpl(numberStore);
```

Comment:
As mentioned in the interface files, if NumberStoreAdditionalOperations inherits from NumberStore, driver code would only need to instantiate one object instead of two.

Reply:
'keyFun' is a significantly different variable name from numberStore, might want to rename one of them for consistency